packages/vm/src/instance.rs

```diff
@@ -9,13 +9,12 @@ use crate::environment::Environment;
 use crate::errors::{CommunicationError, VmError, VmResult};
 use crate::features::required_features_from_wasmer_instance;
 use crate::imports::{
-    native_addr_canonicalize, native_addr_humanize, native_addr_validate, native_db_read,
-    native_db_remove, native_db_write, native_debug, native_ed25519_batch_verify,
-    native_ed25519_verify, native_query_chain, native_secp256k1_recover_pubkey,
-    native_secp256k1_verify,
+    do_write, do_remove, do_addr_validate, do_addr_canonicalize,
+    do_addr_humanize, do_secp256k1_verify, native_debug, do_secp256k1_recover_pubkey,
+    do_ed25519_verify, do_ed25519_batch_verify, do_query_chain,
+    do_scan, do_next, native_db_read
 };
 #[cfg(feature = "iterator")]
-use crate::imports::{native_db_next, native_db_scan};
```

Comment:
Those two were separate because they are only imported when the "iterator" feature is enabled. You need to do the same as before with `do_next` and `do_scan`